execution-scheduler/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup
+
+setup(
+    name='cloudify-execution-scheduler',
+    version='5.2.0.dev1',
+    author='Cloudify',
+    author_email='cosmo-admin@cloudify.co',
+    packages=['execution_scheduler'],
+    entry_points={
+        'console_scripts': [
+            'cloudify-execution-scheduler = execution_scheduler.main:main',
+        ]
+    },
+    install_requires=[
+        'cloudify-common==5.2.0.dev1',
```

Comment:
the scheduler doesnt use common directly, but it uses manager_rest, 
```suggestion
        'cloudify-rest-service==5.2.0.dev1',
```